blacklist/blacklist/Scenes/Debtors/DebtorsPresenter.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  DebtorsPresenter.swift
+//  blacklist
+//
+//  Created by Santiago Carmona Gonzalez on 2/22/18.
+//  Copyright © 2018 Opelty's Open Source Projects. All rights reserved.
+//
+
+import Foundation
+
+protocol DebtorsView: View {
+    func showDebtors(debtors: [Debtor])
+}
+
+class DebtorsPresenter: Presenter {
+    typealias V = DebtorsView
+
+    weak var view: V?
+
+    func loadDebtors() {
+        // TODO: Call Realm or whatever to get the debtors list
+
+        var debtors: [Debtor] = [
```

Comment:
We can try to add some mock data, whatever, the main idea is to populate realm on the next tasks